lighthouse-core/report/v2/report-renderer.js

```diff
@@ -212,19 +292,25 @@ window.ReportRenderer = class ReportRenderer {
    * @return {!Element}
    */
   _renderAudit(audit) {
-    const element = this._createElement('div', 'lighthouse-audit');
-    let title = audit.result.description;
-    if (audit.result.displayValue) {
-      title += ': ' + audit.result.displayValue;
-    }
-    element.appendChild(this._renderScore(audit.score, title, audit.result.helpText));
-    if (audit.result.details) {
-      element.appendChild(this._renderDetails(audit.result.details));
+    const element = this._dom.createElement('div', 'lighthouse-audit');
+    element.appendChild(this._renderAuditScore(audit));
+
+    // Append audit details to to header section so the entire audit is within one <details>.
+    const header = element.querySelector('.lighthouse-score__header');
+    if (header) {
```

Comment:
it was meant to protect against `null` querySelector results like methods. But removed since we don't care about that anymore.